packages/react-components/src/components/FieldError/FieldError.tsx

```
@@ -1,6 +1,6 @@
 import * as React from 'react';
 
-import { Info } from '@livechat/design-system-icons';
+import { Error } from '@livechat/design-system-icons';
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)